_docs/policies/000-submitting-policy.md

```diff
@@ -0,0 +1,36 @@
+# Purpose
+
+Policy Zero establishes the foundational guidelines for creating, reviewing, and maintaining policies in the `status-go` GitHub repository. This policy aims to create a collaborative, inclusive, and transparent process for defining repository policies, specifically regarding how developers engage with and contribute to the repository.
+
+# Submitting a Policy Proposal
+
+- Any individual MAY propose a new policy.
+- Policy ideas SHOULD be discussed with Core Contributors (CCs) and other community members.
```

Comment:
That's not very precise and a bit redundant to "Policy PRs SHOULD be reviewed by as many Core Contributors (CCs) as possible.". I would allow people to submit a policy PR as a forum for discussion.

Reply:
After second though, as it uses SHOULD, I think it is fine to keep it :thinking: